File: backend/src/main/java/com/yagubogu/talk/dto/v1/TalkRequest.java
package com.yagubogu.talk.dto.v1;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record TalkRequest(
        @NotBlank(message = "clientMessageId는 필수입니다")
        String clientMessageId,

        @NotBlank(message = "내용은 필수입니다")
        @Size(max = 500, message = "내용은 500자를 초과할 수 없습니다")
        String content
) {
}

File: android/app/src/main/java/com/yagubogu/data/dto/request/talk/TalkRequest.kt
package com.yagubogu.data.dto.request.talk

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class TalkRequest(
    @SerialName("content")
    val content: String, // 전송할 메시지 내용
)
